packages/web/src/alliance/Affirmations.tsx

```diff
@@ -0,0 +1,152 @@
+          </H2>
+        </Cell>
+      </GridRow>
+      <Exemplar
+        belief={<TransItalic i18nKey="affirmations.laboratoriaBelief" />}
+        copy={t('affirmations.laboratoriaCopy')}
+        image={laboratoria}
+        preview={laboratoriaPreview}
+        logo={<Image resizeMode="contain" source={laboratoriaLogo} style={orgStyle.laboratoria} />}
+        contentStyle={styles.laborStyle}
+      />
+      <Exemplar
+        belief={<TransItalic i18nKey="affirmations.wfpBelief" />}
+        copy={t('affirmations.wfpCopy')}
+        image={wfp}
+        preview={wfpPreview}
+        logo={<Image resizeMode="contain" source={wfpLogo} style={orgStyle.wfpLogo} />}
+        button={{
+          text: t('affirmations.wfpButton'),
+          href:
+            'https://medium.com/celoorg/how-to-design-for-all-stories-from-tanzania-refugees-8b34594d64ae',
+        }}
+        contentStyle={styles.cLabStyle}
+      />
+      <Exemplar
+        belief={<TransItalic i18nKey="affirmations.cLabsBelief" />}
+        copy={t('affirmations.cLabsCopy')}
+        image={clabs}
+        preview={clabsPreview}
+        button={{
+          text: t('affirmations.cLabsButton'),
+          href: 'https://medium.com/celohq/a-cryptocurrency-for-every-juan-144144e62d5',
+        }}
+        logo={<Image resizeMode="contain" source={cLabsLogo} style={orgStyle.cLabsLogo} />}
+        contentStyle={styles.wfpStyle}
+      />
+    </View>
+  )
+}
+
+interface Props {
+  image: ImageSourcePropType
+  preview: ImageSourcePropType
+  logo: React.ReactNode
+  belief: React.ReactNode
+  copy: string
+  contentStyle?: ViewStyle
+  button?: {
+    text: string
+    href: string
+  }
+}
+
+function Exemplar({ image, logo, belief, copy, button, preview, contentStyle }: Props) {
+  const { isMobile } = useScreenSize()
+  return (
+    <GridRow allStyle={standardStyles.elementalMargin}>
+      <Cell span={Spans.half}>
+        <View style={contentStyle}>
+          <H4 style={isMobile ? standardStyles.halfElement : standardStyles.elementalMarginBottom}>
+            {belief}
+          </H4>
+          <View style={isMobile ? standardStyles.halfElement : standardStyles.elementalMargin}>
+            {logo}
+          </View>
+          <Text style={fonts.p}>{copy}</Text>
+          {button && (
+            <Button
+              style={standardStyles.blockMarginTopMobile}
+              kind={BTN.NAKED}
+              size={SIZE.normal}
+              text={button.text}
+              href={button.href}
+            />
+          )}
+        </View>
+      </Cell>
+      <Cell span={Spans.half}>
+        <View style={!isMobile && styles.photoContainer}>
+          <Photo image={image} ratio={471 / 550} preview={preview} />
+        </View>
+      </Cell>
+    </GridRow>
+  )
+}
+
+function TransItalic({ i18nKey }: { i18nKey: string }) {
+  return (
+    <Trans ns={NameSpaces.alliance} i18nKey={i18nKey}>
+      <Text style={textStyles.italic}>""</Text>
```

Comment:
nit: not sure why you're using the Trans wrapper here instead of use `t`

Reply:
with `t` its not possible place different styles on parts of a phrase. IE only Italicize one word in a sentence. with <Trans> Can do it like `i18nkey: "I would like a <0>lot</0> of bread"  and the the 0th child given to <trans> replaces the `<0>`